sql/src/test/java/io/crate/integrationtests/SysCheckerIntegrationTest.java

```diff
@@ -61,6 +57,7 @@ public void testMinimumMasterNodesCheckWithResetCorrectSetting() {
         int setMinimumMasterNodes = numberOfMasterNodes() / 2 + 1;
         execute("set global discovery.zen.minimum_master_nodes=?", new Object[]{setMinimumMasterNodes});
         SQLResponse response = execute("select severity, passed from sys.checks where id=?", new Object[]{1});
+        assertThat(response.rowCount(), equalTo(1L));
```

Comment:
I don't think that having the test fail due to the assertion is much better than having it fail due to another error as long as it fails if something is wrong.

Having a rowCount assertion here implies (at least to me) that this test is testing if the whereClause works correctly. But the purpose of this test is to verify that the calculation for severity/passed works correctly.

As a compromise you could also just output the rows as string and assert the complete result?